slides/lectures/02/mmio/slides.md

```diff
@@ -0,0 +1,216 @@
+---
+layout: section
+---
+# MMIO
+Memory Mapped Input Output
+
+---
+---
+# 8 bit processor
+a simple 8 bit processor with a text display
+
+![8 Bit Processor](/mmio/8-bit-processor.svg)
+---
+layout: two-cols
+---
+# STM32L0x2
+A real MCU
+
+| | |
+|-|-|
+| Cortex-M0+ Peripherals | MCU's *settings* and internal peripherals, available at the same address on all M0+ |
+| Peripherals | GPIO, UART, SPI, I2C, US, etc |
```

Comment:
```suggestion
| Peripherals | GPIO, USART, SPI, I2C, USB, etc |
```

Reply:
USART is specific, usually it is UART, no?

Reply:
I see what you mean, it is the STM32 example, and it is USART. Done.